htdocs/compta/facture/class/facture-rec.class.php

```diff
@@ -86,6 +86,20 @@ class FactureRec extends CommonInvoice
 	 */
 	public $titre;
 
+	/**
+	 * @var int
+	 */
+	public $fk_soc;
+
+	/**
+	 * @var double
+	 */
+	public $remise_percent;
```

Comment:
There is no column remise_percent in table facturerec, so there is no reason to have such a property on this object.